src/utils/audioEngine.js

```
@@ -580,6 +587,10 @@ function stopAudioInternal() {
     part.dispose()
     part = null
   }
+  if (midiParts.length > 0) {
+    midiParts.forEach(trackPart => trackPart.dispose())
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)